### PR TITLE
make workflow config env var and add new NWN config file

### PR DESCRIPTION
**Pull request by MarkTNO**

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -21,4 +21,6 @@ INFLUXDB_WRITE_USER_PASSWORD=write9012
 INFLUXDB_FRONTEND_ADMIN_USER=front1
 INFLUXDB_FRONTEND_ADMIN_USER_PASSWORD=front9012
 
+WORKFLOW_SETTINGS_FILE=./config/workflow_config_nwn_9sept2024.json
```
Comment (cwang39403) [inline .env.template:24]: Perhaps update the instruction in the README about this new env. workflow file path is available

**Review by cwang39403 — APPROVED** [review]

**Comment by lfse-slafleur** [comment]
Looks good! Good idea as well to use an env var.

**Review by lfse-slafleur — APPROVED** [review]

**Comment by lfse-slafleur** [comment]
Also, with https://github.com/Project-OMOTES/omotes-system/pull/96 we introduced a changelog. Feel free to add this change here as well: https://github.com/Project-OMOTES/omotes-system/blob/main/CHANGELOG.md

File: .env.template (hunk 1 above)
Comment (MarkTNO) [reply .env.template:25]: see 0f986c21b6dfe62d35b39de48d5657176acc9e19

**Comment by MarkTNO** [comment]
changelog updated: 0f986c21b6dfe62d35b39de48d5657176acc9e19

**Pull request merged**